subworkflows/local/utils_nfcore_taxprofiler_pipeline/main.nf

```diff
@@ -264,7 +264,8 @@ def toolCitationText() {
         params.perform_shortread_complexityfilter       ? text_shortreadcomplexity : "",
         params.perform_shortread_hostremoval            ? text_shortreadhostremoval : "",
         params.perform_longread_hostremoval             ? text_longreadhostremoval : "",
-        text_classification,
+        [params.run_bracken, params.run_kraken2, params.run_krakenuniq, params.run_metaphlan, params.run_malt, params.run_diamond, params.run_centrifuge, params.run_kaiju, params.run_motus, params.run_ganon, params.run_kmcp].any() ?
```

Comment:
`Text_classification` was defined on line 234. Why was it replaced here?

Reply:
Because of a bit of a edge case:

If you don't run any profiling (as with `test_noprofiling`), the text description ends up with a random half completed sentence:

> 'Performed long read preprocessing with NanoQ (Steinig and coin 2022). Taxprofiling was performed with: . Report generated with MultiQC.'

So I've added this long condition so `text_classificaiton` is only inserted (see new line 268) if any of the profilers are turned on. Otherwise don't insert the beginning of that sentence :)